src/System.Net.HttpListener/src/System/Net/Managed/HttpResponseStream.Managed.cs

```diff
@@ -107,7 +108,30 @@ public override void Close()
             }
         }
 
-        private MemoryStream GetHeaders(bool closing)
+        internal async Task WriteWebSocketHandshakeHeadersAsync()
+        {
+            if (_disposed)
+                throw new ObjectDisposedException(GetType().ToString());
+
+            byte[] bytes = null;
+            MemoryStream ms = GetHeaders(false, true);
```

Comment:
Nit: please add named parameters at the call site, so it's obvious what these Booleans mean